subnet/deployment-generator/script/config_gen.js

```diff
@@ -81,19 +82,25 @@ function configSanityCheck(config){
     process.exit(1)
   }
 
-  if (config.secret_string=='') {
+  if (config.secret_string==='') {
     console.log('SERVICES_SECRET cannot be empty string')
     process.exit(1)
   }
 
-  if (!(config.relayer_mode == 'full' || config.relayer_mode == 'lite')) {
+  if (!(config.relayer_mode === 'full' || config.relayer_mode === 'lite')) {
     console.log('RELAYER_MODE only accepts \'full\' or \'lite\' (default full)')
     process.exit(1)
   }
 
-  if (!(config.parentnet.network == 'devnet'  || 
-        config.parentnet.network == 'testnet' || 
-        config.parentnet.network == 'mainnet' )){
+  if (!(config.parentnet.network === 'devnet'  || 
+        config.parentnet.network === 'testnet' || 
+        config.parentnet.network === 'mainnet' )){
+    var official_urls = {
+      'devnet':'https://devnetstats.apothem.network/devnet'  ,
+      'testnet':'https://erpc.apothem.network/' ,
+      'mainnet': 'https://devnetstats.apothem.network/mainnet' //confirm url
+    }
```

Comment:
this is not what I mean, have a json config put those values in, and read it. Like this way: https://github.com/XinFinOrg/XDC-blockchain-monitor/blob/main/config.json#L27

Reply:
hmm, I don't know, that creates more files. We don't have that many constants and the object is a form of json.